test/instance.ts

```diff
@@ -61,13 +64,14 @@ describe('Instance', () => {
   // tslint:disable-next-line variable-name
   let Instance: typeof inst.Instance;
   let instance: inst.Instance;
+  const sandbox = sinon.createSandbox();
 
-  const SPANNER = {
+  const SPANNER = ({
     request: util.noop,
     requestStream: util.noop,
     projectId: 'project-id',
     instances_: new Map(),
-  };
+  } as {}) as Spanner;
```

Comment:
Do you need the round brackets `()` here?

Reply:
it's added while running `npm run fix`

Reply:
👍